src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -41,118 +23,40 @@ export interface ConversationSpaceProps {
-      </InformationLeftColumn>
-      <VenueWithOverlay venue={venue} containerClassNames="conversation-space">
-        {!seatedAtTable && parentVenueId && parentVenue && (
-          <BackButton variant="simple" space={parentVenue} />
-        )}
-        <div className={`scrollable-area ${seatedAtTable && "at-table"}`}>
-          {venue.description?.text && (
-            <div className="row">
-              <div className="col">
-                <div className="description">
-                  <RenderMarkdown text={venue.description?.text} />
-                </div>
-              </div>
-            </div>
-          )}
-          <div className="container-in-row">
-            <div className="video-wrapper">
-              {seatedAtTable && (
-                <TableHeader
-                  seatedAtTable={seatedAtTable}
-                  setSeatedAtTable={setSeatedAtTable}
-                  venueId={venue.id}
-                  venueName={venue.name}
-                  tables={tables}
-                  defaultTables={CONVERSATION_TABLES}
-                />
-              )}
-              {seatedAtTable && (
-                <div className="participants-container">
-                  <Room
-                    setSeatedAtTable={setSeatedAtTable}
-                    venueId={venue.id}
-                    roomName={`${venue.id}-${seatedAtTable}`}
-                  />
-                </div>
-              )}
-              {!seatedAtTable && (
-                <TablesControlBar
-                  containerClassName="ControlBar__container"
-                  onToggleAvailableTables={toggleTablesVisibility}
-                  showOnlyAvailableTables={showOnlyAvailableTables}
-                />
-              )}
+      <div className={styles.componentSpaceInfo}>
+        {venue.name && <h1>{venue.name}</h1>}
+      </div>
+
+      {venue.description?.text && (
+        <div className="row">
+          <div className="col">
+            <div className="description">
+              <RenderMarkdown text={venue.description?.text} />
             </div>
           </div>
-          {userWithId && (
-            <TableGrid
-              venueId={venue.id}
-              joinMessage={venue.hideVideo === false}
-              customTables={tables}
-              defaultTables={CONVERSATION_TABLES}
-              showOnlyAvailableTables={showOnlyAvailableTables}
-              venue={venue}
-              userId={userWithId.id}
-            />
-          )}
-
-          <UserList
-            usersSample={venue.recentUsersSample ?? ALWAYS_EMPTY_ARRAY}
-            userCount={venue.recentUserCount ?? 0}
-            activity={venue?.activity ?? "here"}
-          />
         </div>
-      </VenueWithOverlay>
+      )}
+
+      <TableGrid
+        venueId={venue.id}
+        joinMessage={venue.hideVideo === false}
+        customTables={tables}
+        defaultTables={CONVERSATION_TABLES}
+        venue={venue}
+        userId={userId}
+      />
```

Comment:
[OOS] Not sure how come `venueId` and `venue` both need to be passed here. A problem for another time maybe

Reply:
removed 'venueId' https://github.com/sparkletown/sparkle/pull/2910/commits/afe06cdd59c668832f14435166c1d3c97fba4064